tsdb/query_executor.go

```diff
@@ -739,6 +739,9 @@ func (q *QueryExecutor) writeInto(row *models.Row, selectstmt *influxql.SelectSt
 	// limitedRowWriter and ExecuteAggregate/Raw makes it ridiculously hard to make sure that the
 	// results will be the same as when queried normally.
 	measurement := intoMeasurement(selectstmt)
+	if measurement == "" {
```

Comment:
`SELECT * into stress."default".:measurement from ...` which carries over the measurement name from the source to the target

It's only really useful if you have a regex measurement source.